pkg/summarizer/summary_test.go

```diff
@@ -1267,35 +1277,181 @@ func TestStatusMessage(t *testing.T) {
 					},
 				},
 			},
-			passingCols:  2, // pass, fail, pass
-			filledCols:   3,
-			passingCells: 6,
-			filledCells:  7,
+			passingCols:       2, // pass, fail, pass
+			filledCols:        3,
+			passingCells:      6,
+			filledCells:       7,
+			expectedFailRatio: []float32{0, 0.33333334, 0},
 		},
 		{
-			name:   "not enough columns yet works just fine",
```

Comment:
Would keep this test case name (I think it's more descriptive of the point of this test, which is that we can summarize even when we don't have all the 'recent' columns filled).

Reply:
Thanks, recreating that test and splitting from 50/50